ghost.py

```diff
@@ -986,6 +995,130 @@ def _get_document(self, key_name):
         return result['hits']['hits'] if result else {}
 
 
+class S3Storage(object):
+    def __init__(self,
+                 db_path=STORAGE_DEFAULT_PATH_MAPPING['s3'],
```

Comment:
If this is required, why provide an entry under `STORAGE_DEFAULT_PATH_MAPPING` for it?